app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -35,7 +34,10 @@
     isPertaxUrl:Boolean
 )(implicit request: Request[_], user: NispUser, partialRetriever: CachedStaticHtmlPartialRetriever)
 
-
+@pageScripts = {
```

Comment:
Since this needs to be in every page, rather than copy and pasting it into every view file we can just add it into main.scala.html